doc/modules/clustering.rst

```diff
@@ -567,7 +567,7 @@ considers at each step all the possible merges.
    number of features. It is a dimensionality reduction tool, see
    :ref:`data_reduction`.
 
-Different linkage type: Ward, complete, average and single linkage
+Different linkage type: Ward, complete, average, and single linkage
 ------------------------------------------------------------------
```

Comment:
Sorry to always come back with a comment, but haven't you forgotten to extend the line below?